auth/user_mgt.go

```diff
@@ -107,83 +97,236 @@ type UserIterator struct {
 
 // UserToCreate is the parameter struct for the CreateUser function.
 type UserToCreate struct {
-	params map[string]interface{}
+	_req   *identitytoolkit.IdentitytoolkitRelyingpartySignupNewUserRequest
+	fields struct {
```

Comment:
Is there a reason these are in a custom struct instead of just at the top level of the UserToCreate struct? If so, maybe a comment here explaining why would be helpful.

Reply:
I suppose the level of indirection can be removed without taking a hit on the readability. Removed.